nft_ingester/src/api/dapi/response.rs

```diff
@@ -23,6 +23,30 @@ pub struct GetGroupingResponse {
     pub group_size: u64,
 }
 
+#[derive(Serialize, Deserialize, Debug, JsonSchema)]
+pub struct HealthCheckResponse {
+    pub status: Status,
+    pub app_version: String,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub node_name: Option<String>,
+    pub checks: Vec<Check>,
+}
+
+#[derive(Serialize, Deserialize, Debug, JsonSchema)]
+pub struct Check {
+    pub status: Status,
+    pub name: String,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub description: Option<String>,
```

Comment:
Can't this field be a part of status? I do not think we need a dedicated description.

Reply:
I added this field down for further expansion.
When we have a check for several services and they can have different errors or problems, it will be easier to read the status from each of them.